dashboard/lib/pd/certificate_renderer.rb

```diff
@@ -0,0 +1,124 @@
+module Pd
+  class CertificateRenderer
+    HARDCODED_CSD_FACILITATOR = 'Dani McAvoy'
```

Comment:
@dmcavoy Do you know if this should be changed?

Reply:
Ya I'm guessing it should be someone else on the Professional Learning Team - maybe Brook?